repo2docker/buildpacks/conda/environment.yml

```diff
@@ -1,9 +1,9 @@
 dependencies:
   - python=3.7.*
-  - ipywidgets==7.2.1
-  - jupyterlab==0.34.9
-  - nbconvert==5.4.0
-  - notebook==5.7.4
+  - ipywidgets==7.4.2
+  - jupyterlab==0.35.4
+  - nbconvert==5.4.1
```

Comment:
are these other packages being bumped because they need to be bumped for the notebook to work?

Reply:
I updated them because they were quite old. Didn't check if we need to. I don't know what our policy is for bumping these dependencies so maybe they shouldn't get changed if it isnn't for security fix reasons (like the notebook).